NuGet.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
```

Comment:
All of these are managed via the `RestoreSources` property in the `Directory.Build.props`